api_return_classes/ApiGame.py

```diff
@@ -0,0 +1,74 @@
+import datetime
+import json
+from enum import Enum, auto
+from typing import List
+
+from api_return_classes.ApiGoal import ApiGoal
+from api_return_classes.ApiTeam import ApiTeam
+from replayanalysis.game.game import Game
+
+
+class ApiGameMap(Enum):
+    Map1 = auto
+
+    @staticmethod
+    def get_map_from_name(name):
+        # TODO: Implement more maps. Maybe as a string literal | thing?
+        # Maps are strings in the pickled game
+        return ApiGameMap.Map1
+
+
+# noinspection PyPep8Naming
+class ApiGameScore:
```

Comment:
can you link to the documentation that created these values.
That way if we ever need to change them we can know the source

Reply:
I've added two commits elaborating on GameMap and replacing the team score's getters to use the team objects' attributes and not the specific game properties thing.